posthog/taxonomy/taxonomy.py

```diff
@@ -1682,6 +1687,80 @@ class CoreFilterDefinition(TypedDict):
             "description": "The name given to this LLM trace, generation, or span.",
             "examples": ["summarize_text"],
         },
+        "$csp_document_url": {
+            "label": "Document URL",
+            "description": "The URL of the document where the violation occurred.",
+            "examples": ["https://example.com/page"],
+        },
+        "$csp_violated_directive": {
+            "label": "Violated directive",
+            "description": "The CSP directive that was violated.",
+            "examples": ["script-src", "img-src", "default-src"],
+        },
+        "$csp_effective_directive": {
+            "label": "Effective directive",
+            "description": "The CSP directive that was effectively violated.",
+            "examples": ["script-src", "img-src", "default-src"],
+        },
+        "$csp_original_policy": {
+            "label": "Original policy",
+            "description": "The CSP policy that was active when the violation occurred.",
+            "examples": ["default-src 'self'; script-src 'self' example.com"],
+        },
+        "$csp_disposition": {
+            "label": "Disposition",
+            "description": "The disposition of the CSP policy that was violated (enforce or report).",
+            "examples": ["enforce", "report"],
+        },
+        "$csp_blocked_url": {
+            "label": "Blocked URL",
+            "description": "The URL that was blocked by the CSP policy.",
+            "examples": ["https://malicious-site.com/script.js"],
+        },
+        "$csp_line_number": {
+            "label": "Line number",
+            "description": "The line number in the source file where the violation occurred.",
+            "examples": ["42"],
+        },
+        "$csp_column_number": {
+            "label": "Column number",
+            "description": "The column number in the source file where the violation occurred.",
+            "examples": ["13"],
+        },
+        "$csp_source_file": {
+            "label": "Source file",
+            "description": "The source file where the violation occurred.",
+            "examples": ["script.js"],
+        },
+        "$csp_status_code": {
+            "label": "Status code",
+            "description": "The HTTP status code that was returned when trying to load the blocked resource.",
+            "examples": ["200", "404"],
+        },
+        "$csp_script_sample": {
+            "label": "Script sample",
+            "description": "A escaped sample of the script that caused the violation. Usually capped at 40 characters.",
+            "examples": ["eval('alert(1)')"],
+        },
+        "$csp_report_type": {
+            "label": "Report type",
+            "description": "The type of CSP report.",
+        },
+        "$csp_raw_report": {
+            "label": "Raw CSP report",
+            "description": "The raw CSP report as received from the browser.",
+            "system": True,
```

Comment:
Why do we have system true here and not for the other ones?

Reply:
It should not be there, good catch